qiita_ware/ebi.py

```diff
@@ -350,9 +355,14 @@ def generate_study_xml(self):
 
         return study_set
 
-    def generate_sample_xml(self):
+    def generate_sample_xml(self, samples=None):
         """Generates the sample XML file
 
+        Parameters
+        ----------
+        samples : list of str, optional
+            The list of samples to be included in the sample xml
```

Comment:
Added

2015-10-07 15:45 GMT-07:00 Yoshiki Vázquez Baeza <notifications@github.com>:

> In qiita_ware/ebi.py
> <https://github.com/biocore/qiita/pull/1489#discussion_r41457729>:
>
> >          """Generates the sample XML file
> >
> > +        Parameters
> > +        ----------
> > +        samples : list of str, optional
> > +            The list of samples to be included in the sample xml
>
> Awesome, thanks - would be great to have an accompanying test case.
> … <#150447b4de91d02f_>
> On (Oct-07-15|15:08), Jose Navas wrote: > """Generates the sample XML file
> > > + Parameters > + ---------- > + samples : list of str, optional > + The
> list of samples to be included in the sample xml Good point about the empty
> list - in case of an empty list or None, all the samples should be used.
> --- Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1489/files#r41454309
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1489/files#r41457729>.
>



-- 
Jose Navas